packages-exp/auth-exp/src/core/credentials/email.test.ts

```diff
@@ -0,0 +1,161 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ProviderId, SignInMethod } from '@firebase/auth-types-exp';
+import { expect, use } from 'chai';
+import * as chaiAsPromised from 'chai-as-promised';
+import { testAuth } from '../../../test/mock_auth';
+import { Auth } from '../../model/auth';
+import { EmailAuthProvider } from '../providers/email';
+import { EmailAuthCredential } from './email';
+import * as mockFetch from '../../../test/mock_fetch';
+import { mockEndpoint } from '../../../test/api/helper';
+import { Endpoint } from '../../api';
+import { APIUserInfo } from '../../api/account_management/account';
+
+use(chaiAsPromised);
+
+describe('core/credentials/email', () => {
+  let auth: Auth;
+  let apiMock: mockFetch.Route;
+  const serverUser: APIUserInfo = {
+    localId: 'local-id'
+  };
+
+  beforeEach(async () => {
+    auth = await testAuth();
+  });
+
+  context('email & password', () => {
+    const credential = new EmailAuthCredential('some-email', 'some-password', EmailAuthProvider.PROVIDER_ID, EmailAuthProvider.EMAIL_PASSWORD_SIGN_IN_METHOD);
```

Comment:
looks like you need to run the prettier

Reply:
do you know how to trigger it manually?

Reply:
`node tools/gitHooks/precommit.js`

It's also at the bottom of the top-level `package.json`